### PR TITLE
Vt only signers execute

### PR DESCRIPTION
**Pull request by verminator23**
Only allow signers / members to execute proposals in cw3-dao and cw3-multisig. Addresses issue #147

### CODE REVIEW

**Comment by JakeHartnell** [comment]
This is great. I think it would be good to have this behind a config option.

Sometimes it's nice to let anyone execute. For example, a DAO could build an auto-execute bot that would automatically execute proposals upon passing.

Think it depends on the DAO / community... maybe @ValarDragon has thoughts on this?

Not sure what the default should be. CC @elsehow / @ezekiiel / @bmorphism,

**Comment by ezekiiel** [comment]
I like the execute bot idea. Am pro putting it behind a config option. Can add a template for this once it lands :)

**Comment by verminator23** [comment]
Great idea. Will add a setting to the config

**Comment by verminator23** [comment]
Made this a config option @ezekiiel @JakeHartnell. I set the default to be true, but lmk if you guys prefer the opposite

File: contracts/cw3-dao/src/contract.rs [hunk 1]
```diff
@@ -59,6 +59,7 @@ pub fn instantiate(
         proposal_deposit: msg.proposal_deposit_amount,
         refund_failed_proposals: msg.refund_failed_proposals,
         image_url: msg.image_url,
+        only_members_execute: true,
```
Comment (JakeHartnell) [inline contracts/cw3-dao/src/contract.rs:62]: Can we update the instantiate msg to include this option?

**Comment by verminator23** [comment]
> One last comment, and then I think this is good to go.

Added it

**Review by JakeHartnell — APPROVED** [review]
LGTM!

**Pull request merged**